spec/features/change_parole_review_date_spec.rb

```diff
@@ -18,9 +18,9 @@
 
     click_link 'Update'
 
-    fill_in 'parole_review_date_form_parole_review_date_dd', with: 13
-    fill_in 'parole_review_date_form_parole_review_date_mm', with: 5
-    fill_in 'parole_review_date_form_parole_review_date_yyyy', with: year
+    fill_in id: 'parole_review_date_form_parole_review_date_3i', with: 13
+    fill_in id: 'parole_review_date_form_parole_review_date_2i', with: 5
+    fill_in id: 'parole_review_date_form_parole_review_date_1i', with: year
 
```

Comment:
What does 3i, 2i, 1i represent?

Reply:
Just day, month, year (3i, 2i and 1i respectively)